src/Framework/PaymentGateways/Contracts/PaymentGatewayInterface.php

```diff
@@ -13,12 +13,13 @@
  */
 interface PaymentGatewayInterface extends SubscriptionModuleInterface
 {
+
     /**
-     * Return a unique identifier for the gateway
+     * Specify which gateway api versions the gateway supports
      *
-     * @since 2.18.0
+     * @unreleased
      */
-    public static function id(): string;
```

Comment:
Why are we getting rid of the `id` function?

Reply:
I think that was an accident